app/assets/stylesheets/custom.scss

```diff
@@ -1,29 +1,35 @@
 .home-search {
+  text-align: center;
+  background:rgba(0,0,0,0.5);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space